icon_manager.py

```diff
@@ -72,10 +98,50 @@ def load():
     if background:
         return
     # Read icons from filepath and store as python data(batch?) for easy access
-    filepath = Path(Path(__file__).parent, "ressources/icons")
+    filepath = get_folder_path()
+
+    for icon in filepath.glob("*.dat"):
+        load_icon(icon)
+
+
+def load_constraint_icons():
+    global constraint_icons
+
+    if constraint_icons:
+        return
+
+    constraint_icons = bpy.utils.previews.new()
+
+    for operator, type in _operator_types.items():
+        icon_path = get_folder_path() / f"{type}.png"
+
+        if not icon_path.exists():
+            continue
+
+        constraint_icons.load(operator, str(icon_path), 'IMAGE')

```

Comment:
Actually kinda strange to depend on rhe operators, couldn't we just convert all files in rhe folder and name the .dat's according to the filename?

Reply:
It's deterministic - we got bitten enough by magic, at this point I'd rather be explicit and correct, haha. Plus, loading PNG files is actually fairly heavy, so if there were eg extras in the folder that we wouldn't want to load, this would be a bad idea anyway.

Reply:
Turns out it _was_ already expensive enough to load the PNGs, I had to load the preview icons instead since they're more than big enough ; used to be 120ms per image, now is 9 (not a typo)